test/test_jit.py

```diff
@@ -1864,10 +1864,6 @@ def forward(self, x):
                    .check("aten::max") \
                    .check("aten::min") \
                    .check("aten::mean") \
-                   .check("aten::__interpolate") \
```

Comment:
@eellison we need the tests as well, actually I think this might have broken these tests, will sync with you tomorrow.